server/src/main/java/org/eclipse/openvsx/LocalRegistryService.java

```diff
@@ -181,6 +182,22 @@ public ResponseEntity<byte[]> getFile(String namespace, String extensionName, St
         return storageUtil.getFileResponse(resource);
     }
 
+    public boolean isType (String fileName){
+        var expectedTypes = Arrays.asList("MANIFEST", "README", "LICENSE", "ICON", "DOWNLOAD", "CHANGELOG");
```

Comment:
you can use `List.of(FileResource.MANIFEST, FileResource.README ... );` here.
`FileResource` is statically imported, so you can just use `MANIFEST`, `README`, etc. Just added `FileResource.` for clarity.